tools/warming_levels_approach.ipynb

```diff
@@ -0,0 +1,952 @@
+    "\n",
+    "plt.title('Comparing log-scales of excessive warming days by different methods')\n",
+    "plt.legend()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "23c585fa-f22d-4522-9972-611b95bcf8a8",
+   "metadata": {},
+   "source": [
+    "Clearly, we can still see the differences between the log-scaled versions of these two distributions. However, if we continue further and fit `pearson3` models to both of these distributions, we'll be able to quantitative discern the differences between the distributions and their confidence intervals."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85371dea-c527-4ae0-bb70-58c4e0ff13e0",
+   "metadata": {},
+   "source": [
+    "#### Fitting `pearson3` distributions to new and traditional methods"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9cdc3ec6-f5d9-46ad-beb0-fc24c308b13e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Importing `pearson3` model fitting function and `threshold_tools` to use distribution on our data.\n",
+    "from scipy.stats import pearson3\n",
+    "from climakitae.explore import threshold_tools"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5c240b3-bfac-4dc9-90fe-b403c432fc11",
+   "metadata": {},
+   "source": [
+    "To begin, we're going to **remove 0's from both distributions** to examine and fit extreme values more closely."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce2e13c8-7c2a-401a-b8e1-11da1f52406d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Removing 0's to examine extreme values\n",
+    "gwl_dist_vals = gwl_dist.to_numpy().flatten()\n",
+    "gwl_dist_vals = gwl_dist_vals[gwl_dist_vals > 0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "db72148a-9f47-4d8c-b3b9-a18984bd8af6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time_period_vals = time_period_dist.to_numpy().flatten()\n",
+    "time_period_vals = time_period_vals[time_period_vals > 0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "863b56fd-2a08-4724-961b-fbd00bf173f7",
+   "metadata": {},
+   "source": [
+    "Now, we're going to fit the datasets to separate `pearson3` distributions, and overlaying the respective distributions and their datasets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2cf19b8b-b98a-469c-944d-0951a882720c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_params, trad_params = fit_models_and_plots(np.log1p(gwl_dist_vals), np.log1p(time_period_vals), 'pearson3');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac5d8ce1-b887-42f6-a7af-51ea8dcb91a8",
+   "metadata": {},
+   "source": [
+    "We can see that there is significant overlap between the two probability distributions of Extreme Heat Days counts, but it is also clear that there is a difference between the two distributions' confidence intervals."
```

Comment:
The `fit_models_and_plots` function gives this figure:
![Screenshot 2023-11-14 095430](https://github.com/cal-adapt/cae-notebooks/assets/56141689/956ed014-8295-4f8f-bda7-e3ae58703b5c)

I think It would be helpful to include one or two sentences on the interpretation of the 95% CI lines, as well as how the figure shows that GWL is more effective than the traditional method.